cmd/update/interfaces/updater.go

```diff
@@ -0,0 +1,14 @@
+package interfaces
+
+import (
+	"context"
+
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
+)
+
+//go:generate mockery -name=Updater -case=underscore
+
+type Updater interface {
+	UpdateNamedEntity(ctx context.Context, name string, project string, domain string, rsType core.ResourceType, cmdCtx cmdCore.CommandContext) error
```

Comment:
not sure if you already know this, but fyi for brevity you can also do `UpdateNamedEntity(ctx context.Context, name, project, domain string, rsType core.ResourceType, cmdCtx cmdCore.CommandContext) error`

Reply:
Sure . Done. Thanks for pointing that out